### DISCUSSION TITLE
Tracking video - repo over its data quote

### DISCUSSION THREAD

**Discussion by SaraGina** [git-lfs/git-lfs#4930] [Q&A]
Hello,
it is the first time I am installing git lfs to upload a video on my repository which is a "reveal.js" presentation. 
When I try to execute the first push to the repo, after having tracked the ".mp4" by git lfs, I get the following message:

batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.               
Uploading LFS objects:   0% (0/1), 0 B | 0 B/s, done.

Could anyone help me with that? I guess if I use git ignore .mp4, git lfs will not work...

Thank you.

**Comment by bk2204** [comment]
Hey,

The file is in your repository now, and ignoring it won't help, because `.gitignore` has no effect on already tracked files.  If you removed the video from the entire history of your repository, then that would help here.  If it's just the latest commit, you can use `git rm` on the file; otherwise, you may need to use `git rebase -x` or `git filter-branch`.

As far as the repository being over its data quota, that's a question you'll have to contact GitHub Support about.  Git LFS knows nothing about data quotas: they're merely an implementation detail of some hosting platforms.
